msal/tests/Test.MSAL.NET.Unit.net45/CacheV2Tests/TokenCacheV2Tests.cs

```diff
@@ -0,0 +1,126 @@
+﻿// ------------------------------------------------------------------------------
+// 
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+// 
+// This code is licensed under the MIT License.
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// 
+// ------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Identity.Client.CacheV2;
+using Microsoft.Identity.Client.CacheV2.Impl;
+using Microsoft.Identity.Client.CacheV2.Impl.InMemory;
+using Microsoft.Identity.Client.CacheV2.Impl.Utils;
+using Microsoft.Identity.Client.CacheV2.Schema;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Core.Instance;
+using Microsoft.Identity.Core.Telemetry;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.Microsoft.Identity.Core.Unit.Mocks;
+
+namespace Test.MSAL.NET.Unit.net45.CacheV2Tests
+{
+    [TestClass]
+    public class TokenCacheV2Tests
+    {
+        private InMemoryCachePathStorage _cachePathStorage;
+        private FileSystemCredentialPathManager _credentialPathManager;
+        private StorageManager _storageManager;
+        private PathStorageWorker _storageWorker;
+        private TokenCacheV2 _tokenCache;
```

Comment:
consider if these should really be part of the class or if the test itself should have an instance. as there is only one test right now I suppose it doesn't matter... but perhaps later it will.

Reply:
Yes, I expect this test class to change a bit as I roll in the rest of the tests.  I'm going to not fix this now but I'll keep it in mind as I bring in the rest of the test coverage here.